lib/galaxy/tools/deps/resolvers/conda.py

```diff
@@ -31,7 +31,7 @@
 
 DEFAULT_BASE_PATH_DIRECTORY = "_conda"
 DEFAULT_CONDARC_OVERRIDE = "_condarc"
-DEFAULT_ENSURE_CHANNELS = "conda-forge,r,bioconda,iuc"
+DEFAULT_ENSURE_CHANNELS = "iuc,bioconda,r,defaults,conda-forge"
```

Comment:
@gregvonkuster if you do `conda config --add channels gregvonkuster` does the channel order change? there's a bit of documentation here: http://conda.pydata.org/docs/channels.html, which also mentions changes in conda 4.1
The commandline / .condarc settings are working fine for me.